blog/2024/04/23/howto-use-official-opensuse-images-with-kanku/index.md

```diff
@@ -0,0 +1,200 @@
+---
+status: published
+title: HowTo use 'official' openSUSE Images with kanku
+date: 2024-04-24 09:40:24
+tags:
+  - howto 
+  - KankuFile
+  - openSUSE
+author: M0ses <fschreiner@suse.de>
+---
+
+In this blog post we will describe the steps to create a KankuFile using a 'official' openSUSE Image.
+
+**TL;DR**
+
+SEE our [example KankuFile](https://raw.githubusercontent.com/M0ses/kanku/master/KankuFile.examples/KankuFile.openSUSE-Leap-15.5-official)
+
+---
+
+## Choose an image
+
+First we would like to explain what settings are required in a KankuFile for a `kanku up` to succeed.
+
+* Access via serial console
+* Configured root login (login_user/login_pass)
+* `installation` section in `Kanku::Handler::CreateDomain` (if a interactive bootstrapping is required - e.g. `firstboot`)
+
+One of design principals of kanku is the old UNIX pragma `KISS` - "Keep It Simple, Stupid".
+As we don`t like wasting time by downloading useless stuff 
```

Comment:
perhaps better to use a normal apostrophe character `'` here instead of the tilde, since the tilde has a special meaning in Markdown... Here the tilde is OK, but if we added another tilde on the same line, Markdown will render the text between the tilde's as "code".

Reply:
fixed them all